yarn.lock

```diff
@@ -6136,7 +6183,7 @@ babel-plugin-lazy-import@arcanis/babel-plugin-lazy-import:
     "@babel/template": "npm:^7.4.4"
   peerDependencies:
     "@babel/core": ^7
-  checksum: e4f81cff69c0704d176b8f80fbe756643d013bcb20e8664164d6d6d943b3f977a0d90d0cc76f1697ccf4e01c3b5a2debc3a1f5ba68fe48e7a680db67ecbfb3a6
```

Comment:
That's weird ... 🤔 I can understand why it crashed (the lockfile entry got generated before the commit hash were added to the resolution field, so it fetched a different commit and thus had a different build), but I don't understand why it would fail now (last commit was 16 days!).

Reply:
Yep, that's weird